pros/conductor/project/__init__.py

```diff
@@ -226,8 +226,17 @@ def make(self, build_args: List[str]):
             make_cmd = 'make'
         stdout_pipe = EchoPipe()
         stderr_pipe = EchoPipe(err=True)
-        process = subprocess.Popen(executable=make_cmd, args=[make_cmd, *build_args], cwd=self.directory, env=env,
+        process=None
+        try:
+            process = subprocess.Popen(executable=make_cmd, args=[make_cmd, *build_args], cwd=self.directory, env=env,
                                    stdout=stdout_pipe, stderr=stderr_pipe)
+        except Exception as e:
+            ui.logger(__name__).warn("\nERROR WHILE CALLING \'" + make_cmd + ".exe\' WITH EXCEPTION \'"+str(e)+"\'.")
```

Comment:
```suggestion
            ui.logger(__name__).warn(f"\nERROR WHILE CALLING {make_cmd}.exe WITH EXCEPTION {str(e)}")
```
Use a format string here

Reply:
you can use error, iirc we have a way to suppress the sentry report prompt